thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashParams.js

```diff
@@ -96,10 +97,11 @@ HashParams.prototype = {
       paramNamesToDefaultValuesMap[this.ANOMALIES_START_DATE] = undefined;
       paramNamesToDefaultValuesMap[this.ANOMALIES_END_DATE] = undefined;
```

Comment:
The model currently has a hard check for `!= undefined`. I'll add refactoring this old code to my to do list.